vividus-to-xray-exporter/src/main/java/org/vividus/xray/facade/XrayFacade.java

```diff
@@ -147,6 +148,21 @@ private void checkIfIssueEditable(String issueKey)
     public void createTestsLink(String testCaseId, String requirementId) throws IOException, JiraConfigurationException
     {
         String linkType = "Tests";
+        JiraEntity issue = jiraFacade.getIssue(testCaseId);
```

Comment:
is this call needed? don't we have testCase instance?

Reply:
discussed